packages/reports/src/Components/Form/CompositeCalendar/two-month-picker.tsx

```diff
@@ -64,9 +64,8 @@ const TwoMonthPicker = React.memo(({ onChange, isPeriodDisabled, value }: TTwoMo
     };
 
     const jumpToCurrentMonth = () => {
-        const current_month = toMoment().endOf('month');
-        setLeftPaneDate(toMoment(current_month).endOf('month').subtract(1, 'month'));
-        setRightPaneDate(current_month);
+        setLeftPaneDate(toMoment().subtract(1, 'month'));
+        setRightPaneDate(toMoment());
```

Comment:
The fix is here: The issue was happening because `.subtract()` is a mutating method. It should be used on a clone or another instance of `toMoment()`.